crates/sui-core/src/execution_scheduler/mod.rs

```diff
@@ -47,16 +47,14 @@ pub enum SchedulingSource {
 pub struct PendingCertificate {
     // Certified transaction to be executed.
     pub certificate: VerifiedExecutableTransaction,
-    // When executing from checkpoint, the certified effects digest is provided, so that forks can
-    // be detected prior to committing the transaction.
-    pub expected_effects_digest: Option<TransactionEffectsDigest>,
+    // Assigned shared object versions and expected effects digest.
```

Comment:
"Just my opinion": if all this comment is going to do is restate the contents of `ExecutionEnv`, I would delete it